### PR TITLE
IDEMPIERE-6110 New Error Message

### PR DESCRIPTION
**Pull request by VitorHenri**
https://idempiere.atlassian.net/browse/IDEMPIERE-6110

# Pull Request Checklist

- [X] My code follows the [code guidelines](https://wiki.idempiere.org/en/Contributing_to_Trunk) of this project
- [X] My code follows the best practices of this project
- [X] I have performed a self-review of my own code
- [X] My code is easy to understand and review. 
- [X] I have checked my code and corrected any misspellings
- [ ] In hard-to-understand areas, I have commented my code.
- [X] My changes generate no new warnings
### Tests
- [X] I have tested the direct scenario that my code is solving
- [ ] I checked all collaterals that can be affected by my changes, and tested other potential affected scenarios
- [ ] New and existing unit tests pass locally with my changes
- [ ] I have added unit tests that prove my fix is effective or that my feature works
### Documentation
- [ ] I have made corresponding changes to the documentation as follows:
- - [ ] New feature (non-breaking change which adds functionality): I have created the New Feature page in the project wiki explaining the functionality and how to use it. If relevant, I have committed sample data to the core seed to have usable examples in GardenWorld.
- - [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected): I have documented the change in a clear way that everyone in the community can understand the impact of the change.
- - [ ] Improvement (improves and existing functionality): This documentation is needed if the improvement changes the way the user interacts with the system or the outcome of a process/task changes. If it is just, for instance, a performance improvement, documentation might not be needed. 
- [ ] The changed/added documentation is in the project wiki (not privately-hosted pdf files or links pointing to a company website) and is complete and self-explanatory.

### CODE REVIEW

**Comment by VitorHenri** [comment]
Could this be included in version 11 please?

**Comment by CarlosRuiz-globalqss** [comment]
please check my comment at
https://idempiere.atlassian.net/browse/IDEMPIERE-6110?focusedCommentId=50745

File: migration/iD11/postgresql/202404171125_IDEMPIERE-6110.sql [hunk 1]
```diff
@@ -0,0 +1,7 @@
+-- IDEMPIERE-6110
+SELECT register_migration_script('202404171125_IDEMPIERE-6110.sql') FROM dual;
+
+-- Apr 17, 2024, 11:25:53 AM BRT
+INSERT INTO AD_Message (MsgType,MsgText,AD_Client_ID,AD_Org_ID,IsActive,Created,CreatedBy,Updated,UpdatedBy,AD_Message_ID,Value,EntityType,AD_Message_UU) VALUES ('E','Line with product or charge defined as Freigth Product or Freight Charge in Tenant configuration not allowed.',0,0,'Y',TO_TIMESTAMP('2024-04-17 11:25:53','YYYY-MM-DD HH24:MI:SS'),100,TO_TIMESTAMP('2024-04-17 11:25:53','YYYY-MM-DD HH24:MI:SS'),100,nextidfunc(9,'N'),'FreightOrderLineNotAllowed','D','204c8bb9-d002-4beb-bbc3-1d1a11d7471d')
```
Comment (hengsin) [inline migration/iD11/postgresql/202404171125_IDEMPIERE-6110.sql:5]: suggestion of a probably more informative error message - "Products or charges configured as 'Freight Product' or 'Freight Charge' cannot be added to this order due to the combination of delivery via rule and freight cost rule"

Comment (VitorHenri) [reply migration/iD11/postgresql/202404171125_IDEMPIERE-6110.sql]: Thanks @hengsin, i have made the change.

File: migration/iD11/oracle/202404171125_IDEMPIERE-6110.sql [hunk 2]
```diff
@@ -0,0 +1,10 @@
+-- IDEMPIERE-6110
+SELECT register_migration_script('202404171125_IDEMPIERE-6110.sql') FROM dual;
+
+SET SQLBLANKLINES ON
+SET DEFINE OFF
+
+-- Apr 17, 2024, 11:25:53 AM BRT
+INSERT INTO AD_Message (MsgType,MsgText,AD_Client_ID,AD_Org_ID,IsActive,Created,CreatedBy,Updated,UpdatedBy,AD_Message_ID,Value,EntityType,AD_Message_UU) VALUES ('E','Products or charges configured as ''Freight Product'' or ''Freight Charge'' cannot be added to this order due to the combination of delivery via rule and freight cost rule',0,0,'Y',TO_TIMESTAMP('2024-04-17 11:25:53','YYYY-MM-DD HH24:MI:SS'),100,TO_TIMESTAMP('2024-04-17 11:25:53','YYYY-MM-DD HH24:MI:SS'),100,nextidfunc(9,'N'),'FreightOrderLineNotAllowed','D','204c8bb9-d002-4beb-bbc3-1d1a11d7471d')
```
Comment (CarlosRuiz-globalqss) [inline migration/iD11/oracle/202404171125_IDEMPIERE-6110.sql:8]: Hi @VitorHenri 

I tested this successfully.

However, there is a minor mismatch between oracle and postgresql message, the postgresql message has an extra dot at the end.  Can you please fix that?

**Review by CarlosRuiz-globalqss — CHANGES_REQUESTED** [review]

File: migration/iD11/oracle/202404171125_IDEMPIERE-6110.sql (hunk 2 above)
Comment (VitorHenri) [reply migration/iD11/oracle/202404171125_IDEMPIERE-6110.sql:8]: Hi @CarlosRuiz-globalqss, sure! Adjusted!

**Review by CarlosRuiz-globalqss — APPROVED** [review]

**Pull request merged**